### PR TITLE
WIP - localize most_severe_accidents_table

### PR DESCRIPTION
**Pull request by atalyaalon**
Missing for this PR to be ready - only segments translation

@ziv17 what do you think regarding the localization?
Here it's done during the infographics data creation - however we can also preform it after we extract from cache - 
What do you think is a better way?
After cache - will be a challenge for the texts we join (for example - title with the road segment) - in this case we should = for example - create the title after the cache
Before cache - add the lang param to the cache and perform localization before cache insertion - we'll encounter no such issues.

### CODE REVIEW

**Comment by atalyaalon** [comment]
@idosavion I suggest merging this one and adding the road segments translation table in another PR

File: anyway/infographics_utils.py [hunk 1]
```diff
@@ -1126,7 +1129,7 @@ def get_most_severe_accidents_with_entities(
 
 
 def get_most_severe_accidents_table_title(location_info):
-    return "תאונות בסדר חומרה יורד במקטע " + location_info["road_segment_name"]
+    return _("Accidents in descending order by severity for segment") + ' ' + segment_dictionary[location_info['road_segment_name']]
```
Comment (idosavion) [inline anyway/infographics_utils.py:1132]: " instead of ' (just run black)

File: anyway/infographics_dictionaries.py [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+                          10: _("Overturned vehicle"),
+                          11: _("Skid"),
+                          12: _("Injury of a passenger in a vehicle"),
+                          13: _("A fall from a moving vehicle"),
+                          14: _("Fire"),
+                          15: _("Other"),
+                          17: _("Collision of rear-end to front"),
+                          18: _("Collision of rear-end to side"),
+                          19: _("Collision with an animal"),
+                          20: _("Damage caused by a falling load off a vehicle")}
```
Comment (idosavion) [inline anyway/infographics_dictionaries.py:29]: I think we should localize this a bit later, pretty sure this dictionary and the logic dependent on it shouldn't be changed by the user preferred language, right?

File: anyway/infographics_utils.py [hunk 3]
```diff
@@ -245,7 +248,7 @@ def prepare_table(location_info, start_time, end_time):
         )
         # Add casualties
         for accident in accidents:
-            accident["type"] = accident["accident_type"]
+            accident["type"] = accident_type_dict[accident["accident_type"]]
```
Comment (idosavion) [inline anyway/infographics_utils.py:251]: how can you be certain that the language used to create the dict is the same used here (so you won't get a keyError)?

**Comment by ziv17** [comment]
I prefer doing the localization aafter retrieving from cache. I am thinking of adding a static function for each widget, that will be called with the request params and the text of that widget, and will perform the localization. 
What do you think?

File: anyway/infographics_dictionaries.py [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+
 driver_type_hebrew_dict = dict(
     professional_driver="נהג מקצועי", private_vehicle_driver="נהג פרטי", other_driver="לא ידוע"
 )
 head_on_collisions_comparison_dict = dict(
     head_to_head_collision="התנגשות חזית בחזית", others="אחרות", head_to_head="חזיתיות"
 )
 english_accident_severity_dict = {1: "fatal", 2: "severe", 3: "light"}
+
+accident_type_dict = {1: _("Pedestrian injury"),
```
Comment (ziv17) [inline anyway/infographics_dictionaries.py:11]: Here the `accident_type_dict` contains the result of the call to _(), right?
I think the user of the dict want to issue the call to _() at the time it of access to the dict. Maybe add a static function that will get the value from the dict and issue the call to _()?

File: anyway/infographics_utils.py (hunk 3 above)
Comment (atalyaalon) [reply anyway/infographics_utils.py:251]: You are right, I can't.

**Comment by ziv17** [comment]
@atalyaalon , @idosavion , I prefer to start the work on moving the localization after the cache with an example of one widget with localization that is using the underscore function `_()`. Can you point me to such an example?

**Pull request closed without merging**